### PR TITLE
Yeet

### PR DESCRIPTION
**Pull request by PowerfulBacon**
(3000 Venezuelan Bolívar bounty)

## About The Pull Request

Adds in a 1% to yeet things instead of throw them

## Why It's Good For The Game

Adds in yeet

## Changelog
:cl:
add: Adds in yeet
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Yeet

**Comment by BrumalLake** [comment]
282 commits?

**Comment by PowerfulBacon** [comment]
> 
> 
> 282 commits?

Yes, this feature required a lot of testing and debugging

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -155,7 +155,13 @@
 			return TRUE
 
 	if(thrown_thing)
-		visible_message("<span class='danger'>[src] throws [thrown_thing].</span>", \
+		if(prob(1))
+			visible_message("<span class='hierosay big'>YEET!</span>")
+			visible_message("<span class='danger'>[src] YEETS [thrown_thing].</span>", \
+						"<span class='danger'>You YEET [thrown_thing].</span>")
+			new /obj/effect/temp_visual/yeet(loc)
```
Comment (Cenrus) [inline code/modules/mob/living/carbon/carbon.dm:163]:
```suggestion
			playsound(src, 'sound/effects/yeet.ogg', 50, 1)
			new /obj/effect/temp_visual/yeet(loc)
```

Comment (PowerfulBacon) [reply code/modules/mob/living/carbon/carbon.dm:163]: perfect

**Comment by eeSPee** [comment]
This PR will either make or break the game. I don't think Beestation is ready for this kind of change... I don't think the world is ready either.

File: code/modules/mob/living/carbon/carbon.dm [hunk 2]
```diff
@@ -155,7 +155,14 @@
 			return TRUE
 
 	if(thrown_thing)
-		visible_message("<span class='danger'>[src] throws [thrown_thing].</span>", \
+		if(prob(1))
+			visible_message("<span class='hierosay big'>YEET!</span>")
+			visible_message("<span class='danger'>[src] YEETS [thrown_thing].</span>", \
+						"<span class='danger'>You YEET [thrown_thing].</span>")
```
Comment (eeSPee) [inline code/modules/mob/living/carbon/carbon.dm:161]: Someone will remove this when the word 'YEET' will inevitably be added to the word filter, I just know it!

**Comment by Archanial** [comment]
Make this emote instead :trollface:

**Comment by utilbird** [comment]
Bacon trying to feign extra repo activity 😏

**Comment by PowerfulBacon** [comment]
> Bacon trying to feign extra repo activity 😏

Got to get them +13 points, corgstation was only in the hundreds of thousands😔

File: code/modules/mob/living/carbon/carbon.dm (hunk 1 above)
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/carbon.dm:163]:
```suggestion
			playsound(src, 'sound/effects/yeet.ogg', 50, 1)
			new /obj/effect/temp_visual/yeet(loc)
```

**Comment by PowerfulBacon** [comment]
oka fine there are now less than 200 commits, I hope you are happy

**Comment by Archanial** [comment]
Funny pr and 300 empty commits was also funny but this shouldn't go live on sage.

**Comment by super12pl** [comment]
> Funny pr and 300 empty commits was also funny but this shouldn't go live on sage.

no fun allowed

**Pull request closed without merging**